### ISSUE TITLE
Can't Open Workflow Settings, Roll Automation is not working.

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#365]
In GitLab by @garrysmod367 on Jun 4, 2021, 02:54

I am on the latest version of MIdiQOl, DAE, and Foundry (0.8.6). I have eliminated all other modules as being an issue. I am unable to open the workflow settings at all. When I attempt to use the standard roll button or a macro created from the item, nothing happens. 
Console displays the following warning:

```
midi-qol.js:31 midi-qol |  show item card  Item5e {labels: {…}, apps: {…}, _sheet: null, parent: Actor5e, pack: null, …}apps: {}labels: {activation: "1 Action", target: "", range: "5 / 5", duration: "", recharge: "Recharge [undefined]", …}_sheet: nullabilityMod: (...)actor: (...)collection: (...)collectionName: (...)compendium: (...)createEmbeddedDocuments: (...)data: ItemData {_id: "R5eQPYKSPb51R8aH", name: "(GS) Chromatic Obsidian Greateaxe", type: "weapon", img: "https://www.dndbeyond.com/avatars/15729/439/637487766472213895.png", data: {…}, …}delete: (...)deleteEmbeddedDocuments: (...)documentName: (...)effects: (...)entity: (...)folder: (...)getEmbeddedCollection: (...)getEmbeddedDocument: (...)getRollData: (...)hasAreaTarget: (...)hasAttack: (...)hasDamage: (...)hasLimitedUses: (...)hasPlayerOwner: (...)hasSave: (...)hasTarget: (...)id: (...)img: (...)isEmbedded: (...)isHealing: (...)isOwned: (...)isOwner: (...)isVersatile: (...)limited: (...)link: (...)name: (...)origData: (...)owner: (...)pack: nullparent: Actor5e {overrides: {…}, apps: {…}, _sheet: null, _tokenImages: null, _lastWildcard: null, …}permission: (...)prepareDerivedData: (...)prepareEmbeddedEntities: (...)roll: (...)rollAttack: (...)rollDamage: (...)sheet: (...)transferredEffects: (...)type: (...)update: (...)updateEmbeddedDocuments: (...)uuid: (...)visible: (...)_data: (...)_id: (...)__proto__: Item Actor5e {overrides: {…}, apps: {…}, _sheet: null, _tokenImages: null, _lastWildcard: null, …}apps: {}known_languages: Set(6) {"common", "sylvan", "deep", "infernal", "drow handcode (basic)", …}overrides: {data: {…}}_lastWildcard: null_sheet: null_tokenImages: nullapplyActiveEffects: (...)collection: (...)collectionName: (...)compendium: (...)data: ActorData {_id: "i9lnhPN3LeMutrsF", name: "Nat", type: "character", img: "ddb-imports/Nat_Clothed.jpg", data: {…}, …}documentName: (...)effects: (...)entity: (...)folder: (...)getRollData: (...)hasPlayerOwner: (...)id: (...)img: (...)isEmbedded: (...)isOwner: (...)isPolymorphed: (...)isToken: (...)itemTypes: (...)items: (...)limited: (...)link: (...)name: (...)owner: (...)pack: nullparent: nullpermission: (...)rollAbilitySave: (...)rollAbilityTest: (...)rollDeathSave: (...)rollSkill: (...)sheet: (...)temporaryEffects: (...)token: (...)type: (...)uuid: (...)visible: (...)_data: (...)_id: (...)__proto__: Actor null false Workflow {rollOptions: {…}, actor: Actor5e, item: Item5e, itemId: "R5eQPYKSPb51R8aH", uuid: "Actor.i9lnhPN3LeMutrsF.Item.R5eQPYKSPb51R8aH", …}actor: Actor5e {overrides: {…}, apps: {…}, _sheet: null, _tokenImages: null, _lastWildcard: null, …}advantage: undefinedattackCardData: undefinedcapsLock: falsecurrentState: 0damageCardData: undefineddamageDetail: []disadvantage: undefineddisplayHookId: nulldisplayId: "aiscrzec7o5h85jk"event: MouseEvent {isTrusted: true, screenX: 286, screenY: 953, clientX: 286, clientY: 930, …}failedSaves: Set(1) {Token}hideTags: []hitTargets: Set(1) {Token}isCritical: falseisFumble: falseitem: Item5e {labels: {…}, apps: {…}, _sheet: null, parent: Actor5e, pack: null, …}itemCardData: {}itemId: "R5eQPYKSPb51R8aH"itemLevel: 0noAutoAttack: falsenoAutoDamage: falseonUseCalled: falseotherDamageDetail: []placeTemlateHookId: nullrollOptions: {disKey: undefined, advKey: undefined, versaKey: undefined, critKey: false, fastForward: false, …}saveRequests: {}saveTimeouts: {}saves: Set(0) {}showCard: truespeaker: {scene: "NJ181YxIQTZgMEAX", token: "xT45pgNcmvbBFFeP", actor: "i9lnhPN3LeMutrsF", alias: "Nat"}superSavers: Set(0) {}targets: Set(1) {Token}templateId: nulltokenId: "xT45pgNcmvbBFFeP"uuid: "Actor.i9lnhPN3LeMutrsF.Item.R5eQPYKSPb51R8aH"_id: "aiscrzec7o5h85jk"damageFlavor: (...)hasDAE: (...)id: (...)workflowType: (...)__proto__: Object
warn @ midi-qol.js:31
showItemCard @ itemhandling.js:422
doItemRoll @ itemhandling.js:362
async function (async)
doItemRoll @ itemhandling.js:247
🎁call_wrapper @ libWrapper-wrapper.js:554
🎁CONFIG.Item.entityClass.prototype.roll#0 @ libWrapper-wrapper.js:142
rollItemMacro @ macros.js:58
eval @ VM2787:5
callScriptMacroFunction @ Macros.js:171
renderMacro @ Macros.js:189
executeMacro @ Macros.js:210
_onClickMacro @ custom-hotbar.js:344
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```

![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/ff30055f9c7d0d7be455eb8fb8f840bf8cf4b731be104eed531ad18b3f2bb6a8/image.png)

**Comment by tposney** [comment]
Are you on the latest dnd5e as well?

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 4, 2021, 23:36

Yes I am.

**Comment by tposney** [comment]
This is the second report I've had of this problem. The message you sent is just a warning that is issued when the debug settings are enabled (I think). Are there any actual error messages (change the console filter to error) and see if there is anything there?

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:16

I do apologize. I still had a filter on that was preventing me from seeing the error. The error I get is this:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/1c44c2649d6ce33d08a806517c35b4a099c38933b2bc3f3f3faed4a352bdee66/image.png)

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:18

This is the error I get when attempting to open the workflow settings:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/733abff8bc0ede77d4c84943b1a241d1769dc4a162ce882d5f0cd78a35e8deb8/image.png)

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:21

This is the error thrown if I don't use a macro to trigger the attack but instead use the "Standard Roll" button: 
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/79acf808b9b35bbfa8ef0352d979b7dcf3de72703ab150e0dbd13ceaabc44b58/image.png)

**Comment by tposney** [comment]
Ok, it's almost certainly the playlist that is the problem. I'm guessing
you have no playlists in your world. For the moment, can you create a
playlist, I don;t think it even needs any entries in it, there just has to
be one.

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:34

I don't believe that is it. I have plenty of playlists:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/17b28f1155bf4cc9545d914efd52ad0fda717f5e8e64dd0fd810a57c93b49e53/image.png)

**Comment by tposney** [comment]
Can you confirm which version of midi-qol you are on? And which foundry
version? That code should not exist anymore.

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:42

Foundry 0.8.6:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/cf15e31aa961e864a5bac02551d2722b080a6d3d2908e4315fbc1c4769cd3c95/image.png)

MidiQOL 0.8.6:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/f900d9d958a49a5cdb25e1245d8f5e2f003d2771739943bcd3ad4fd8af7e68b6/image.png)

DND 5e 1.3.3:
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/5480d3390166eecc623bbaa7e58fcf150e7493fbde1dd5c13d7b09a926cd1bb5/image.png)

**Comment by tposney** [comment]
In GitLab by @garrysmod367 on Jun 5, 2021, 24:50

I wanted to be sure there is nothing else conflicting so I made a new world with only these three modules loaded: libwrapper, socketlab, and MidiQOL. I created only one actor and only equipped with an item from the SRD Items that come with the DND 5e system and the same error occurs.

I also tried it without LibWrapper just to be extra safe but still nothing. I also still can't open workflow settings.
